### ISSUE TITLE
Content overlapping in the bottom

### ISSUE THREAD

**Issue by Raunaksingh100** [fossasia/fossasia.org#810]
### Description-
content overlapping in the bottom side in the mobile view see in the screenshot given below.

### Screenshot-
![bug](https://user-images.githubusercontent.com/65535360/104043510-f1cb2780-5201-11eb-9f3e-1de02a788501.PNG)

**Comment by codewithnick** [comment]
Hey, how can i contribute to solve this ,please tell me how do i reproduce the bug

**Comment by Raunaksingh100** [comment]
> Hey, how can i contribute to solve this ,please tell me how do i reproduce the bug.


Go to codeheat website and see the bug

**Comment by codewithnick** [comment]
> > Hey, how can i contribute to solve this ,please tell me how do i reproduce the bug.
> 
> 
> Go to codeheat website and see the bug

Haven't participated in codeheat yet..should I participate first.

**Comment by Raunaksingh100** [comment]
@codewithnick This issue not valid for "Codeheat" You need to fixed the "Codeheat label issue" only for codeheat issue contest
for more details visit the codeheat.org website

**Comment by codewithnick** [comment]
Yes I know this contribution won't be counted in codeheat but I still want to make my first contribution here..please assign me the issue

**Comment by ravikr126** [comment]
Hii I would like to contribute in this issue.please assign this issue to me
